Directory.Build.props

```diff
@@ -28,7 +28,7 @@
     -->   
     <NoWarn>$(NoWarn);MSB3270</NoWarn>
     <WarningsAsErrors>true</WarningsAsErrors>
-    <LangVersion>10.0</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
Is changing this to preview going to break anything else?

Reply:
It won't break any of our other repos. And we can update this to 11 in a few weeks once things get shipped.